recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
   run:
     - python
     - numpy
-    - gdal 2.2.*
+    - gdal 2.1.*
```

Comment:
Yes `rios` runs fine with `gdal` 1.x. We did discuss the pinnings here: https://github.com/conda-forge/rios-feedstock/pull/9. I'm happy to remove the pinning and see what happens.